src/Extensions/Shopware/DataGenerator/Resources/Categories.php

```diff
@@ -38,7 +38,7 @@ class Categories extends BaseResource
      */
     private function buildNestedTree($number, $parentCategory = null, $depth = 0, $leftNeighbour = null)
     {
-        $categoriesOnThisLevel = ($depth === 1) ? 6 : rand(2, 10);
+        $categoriesOnThisLevel = ($depth === 0) ? 6 : rand(2, 10);
```

Comment:
First level ist defined over `$depth = 0` this generate always a top navigation with six categories